vault/logical_system.go

```diff
@@ -1895,6 +1963,13 @@ func (b *SystemBackend) handleEnableAuth(ctx context.Context, req *logical.Reque
 	var apiConfig APIMountConfig
 
 	configMap := data.Get("config").(map[string]interface{})
+	// Augmenting configMap for some config options to treat them as comma separated entries
+	err := augmentEnableAuthConfigMap(configMap)
+	if err != nil {
+		return logical.ErrorResponse(
```

Comment:
Typically when returning an ErrorResponse we return a nil error and let the framework sort out what HTTP error code to return.  It's not wrong to do it this way though.

Reply:
I see, I just tried to be consistent with the use of logical.ErrorResponse in this file.

Reply:
Yup, fine either way.